gui/src/styles-dashboard-workspace.css

```
@@ -527,3 +527,164 @@
   color: var(--text);
   white-space: nowrap;
 }
+
+
+/* Overview settings share a fixed control column, then stack together when the
+   content area is too narrow. Scope this to Overview; provider/model tables keep
+   their own layout. Dropdowns and popovers already portal outside the container. */
+.dash-overview-stack {
+  --dash-controls-width: 26rem;
+  container: dashboard-overview / inline-size;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)